src/components/Explorer/fields.js

```diff
@@ -492,7 +509,10 @@ ${props.player && props.player.value ? '' : 'AND player_matches.account_id < pla
       const laneRoleId = Number(str.substring('lane_role_'.length));
       return { text: strings[str], value: laneRoleId, key: String(laneRoleId) };
     }),
-    isTi8Team: [{ text: 'Yes', value: true, key: 'true' }],
+    isTi8Team: [{ text: '\u2714', value: true, key: 'true' }],
```

Comment:
what is \u2714?

Reply:
it's a check mark

Reply:
I would recommend text since it's more understandable